help.txt

```diff
@@ -1,33 +1,32 @@
 MANAGE PLUGINS
-  asdf plugin add <name> [<git-url>]       Add a plugin from the plugin repo OR, add a Git repo
-                                           as a plugin by specifying the name and repo url
-  asdf plugin list [--urls] [--refs]       List installed plugins. Optionally show git urls and git-ref.
-  asdf plugin list all                     List plugins registered on asdf-plugins repository with URLs
-  asdf plugin remove <name>                Remove plugin and package versions
-  asdf plugin update <name> [<git-ref>]    Update a plugin to latest commit or a particular git-ref.
-  asdf plugin update --all                 Update all plugins
+asdf plugin add <name> [<git-url>]      Add a plugin from the plugin repo OR, add a Git repo as a plugin by specifying the name and repo url
```

Comment:
I think we should still wrap this text at 80 columns. I work on pretty small screens and my terminal is usually around 80 columns.